pkg/ebpf/tracer.go

```diff
@@ -284,16 +287,18 @@ func (t *Tracer) initPerfPolling() (*bpflib.PerfMap, error) {
 	go func() {
 		// Stats about how much connections have been closed / lost
 		ticker := time.NewTicker(5 * time.Minute)
+		flushIdle := time.NewTicker(20 * time.Second)
 
 		for {
 			select {
-			case rawConns, ok := <-closedChannel:
+			case batchData, ok := <-closedChannel:
 				if !ok {
-					log.Infof("Exiting closed connections polling")
 					return
 				}
 				atomic.AddInt64(&t.perfReceived, 1)
-				conns := decodeRawTCPConns(rawConns)
+
+				batch := newBatch(batchData)
+				conns := t.batchManager.Extract(batch, time.Now())
```

Comment:
If those calls to `time.Now()` show up in the CPU profile, I'll create a "throttled" timer